### PR TITLE
Shawns branch

### PR DESCRIPTION
**Pull request by tubashawn**

### CODE REVIEW

File: rent_app/client/src/pages/Landing/Landing.js [hunk 1]
```diff
@@ -29,8 +29,12 @@ class Landing extends Component {
                     </Link>
                 </Col>
             </Container>
+            // <div>
```
Comment (mikeytdan) [inline rent_app/client/src/pages/Landing/Landing.js]: Is this still needed?

File: rent_app/client/src/components/Unit/Unit.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    };
+
+    handleInputChange = event => {
+        const { name, value } = event.target;
+        this.setState({
+            [name]: value
+        });
+    };
+
+    // handleFormSubmit = event => {
```
Comment (mikeytdan) [inline rent_app/client/src/components/Unit/Unit.js]: Is this code needed?

File: rent_app/client/src/pages/Register/Register.js [hunk 3]
```diff
@@ -7,7 +7,17 @@ class Register extends Component {
...
+        businessPhone: "",
+        cellPhone: "",
+        homePhone: "",
+        fax: "",
+        businessAddress: "",
+        mailingAddress: "",
+        homeAddress: ""
     };
 
     // compnonentDidMount() {
```
Comment (mikeytdan) [inline rent_app/client/src/pages/Register/Register.js]: Is this code needed?

File: rent_app/client/src/pages/Landing/Landing.js (hunk 1 above)
Comment (MCPolichette) [reply rent_app/client/src/pages/Landing/Landing.js]: I agree, I don't think we need anything outside of the container.

**Pull request merged**